src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -11,6 +10,7 @@
 import java.util.Date;
 import java.util.HashMap;
 import java.util.Iterator;
+import java.util.LinkedList;
```

Comment:
Are there any added benefits for using linked list instead of the current use of arraylist?

If there's none, let's stick to arraylist instead.

Reply:
There is no particular reason for using linked list. I've replaced the usages of linked list with arraylist.